source/documentation/rdo-packaging-guidelines.html.md

```diff
@@ -48,8 +55,22 @@ on the ticket.
 ```bash
 Restart=[on-failure|always]
 ```
+* If a service depends on other for proper start, you can use the `After=` and
+optionally `Requires=` parameters in unit configuration file. An example of
+dependencies for neutron-openvswitch-agent service:
+
+```bash
+After=syslog.target network.target network.service openvswitch.service
+Requires=openvswitch.service
+```
+
+* The systemd package provides a set of rpm macros to handle systemd operations
+on %post, %preun and %postun (more details [here](https://fedoraproject.org/wiki/Packaging:Scriptlets#Systemd)).
+
+### python packaging guidelines
 
-### General guidelines
+* Some recommendations about python packaging can be found in
```

Comment:
Redundant w/ the above statement that we are based upon Fedora packaging guidelines, the goal of this document is to provide specific RDO guidelines or exceptions to the Fedora ones (which are themselves the foundation of RHEL's guidelines)